app/javascript/mastodon/locales/zh-TW.json

```diff
@@ -29,6 +29,9 @@
   "column_back_button.label": "上一頁",
   "column_subheading.navigation": "瀏覽",
   "column_subheading.settings": "設定",
+  "community.column_settings.advanced": "進階",
```

Comment:
Some of these have been already changed according to the type of the timelines. For example, please see pt.json(Portuguese) and pt-BR.json(Portuguese in Brazil), It is set as following.
```"home.settings": "Parâmetros da listagem Home"```
and I set community and public as following.
```"community.settings": "Parâmetros da listagem Local"```
```"public.settings": "Parâmetros da listagem Global"```
Therefore, I made the original keys for such individual changes in each languages.